<launch>

  <arg name="nodelet_manager" default="lidar_start_nodelet_manager"/>
  <node pkg="nodelet"  type="nodelet" name="$(arg nodelet_manager)" args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="LidarStartNodelet" args="load lidar_start/LidarStartNodelet $(arg nodelet_manager)"  output="screen">
    <!-- LiDAR params -->
    <param name="lidar_type" value="RSL32"/> <!-- P40P RSL32 VLP16 is now avaiable -->
    <param name="ip" value="192.168.1.200"/>
    <param name="data_port" value="6699"/>
    <param name="direction" value="3"/>
    <param name="mode" value="0"/>
    <param name="version" value="0"/>
    <param name="RS32_angle_correction_file_path" value="$(find lidar)/param/RS32/NP19/right/angle.csv"/>
    <param name="RS32_ChannelNum_correction_file_path" value="$(find lidar)/param/RS32/NP19/right/ChannelNum.csv"/>
    <param name="RS32_CurveRate_correction_file_path" value="$(find lidar)/param/RS32/NP19/right/CurveRate.csv"/>
    <param name="RS32_curves_correction_file_path" value="$(find lidar)/param/RS32/NP19/right/curves.csv"/>
    <!-- Topic Setting -->
    <param name="frame_id" value="rslidar"/>
    <param name="origin_cloud_topic" value="right_points"/>
  </node>

</launch>
